src/SFA.DAS.Commitments.Api/Controllers/TransferController.cs

```diff
@@ -27,7 +27,6 @@ public TransferController(IEmployerOrchestrator employerOrchestrator)
             return Ok(response);
         }
 
-        [Obsolete]
```

Comment:
Why are we removing [Obsolete]?

Reply:
The reason I was looking at the methods marked [Obsolete], was because they were emitting warnings, and I have been gradually fixing warnings and switching on TreatWarningsAsErrors in (warning-free) projects.
The last remaining warnings in the solution were all related to the use of [Obsolete] methods, and now that the remaining [Obsolete] attributes have been removed, *all* projects now treat warnings as errors.

This action, in particular, *is* currently used and I couldn't find any other action that could be used instead.
I checked to see if the current consumer could be switched over to using the existing GetCommitment action in the EmployerController, but the TransferController's version ultimately checks that the caller's Id matches the SendingEmployerAccountId in the transfer request, as opposed to the ReceivingEmployerAccountId.